src/app/features/characters/components/find-character-form/find-character-form.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnDestroy } from '@angular/core';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { Router } from '@angular/router';
+import { MarvelChar } from '@characters/models/characters.model';
+import { CharactersMarvelService } from '@characters/services/characters-marvel.service';
+import { Subscription } from 'rxjs';
+import { CharacterSearchStatusEnum } from '@characters/models/characters.enum';
+@Component({
+  selector: 'find-character-form',
+  standalone: true,
+  imports: [ ReactiveFormsModule ],
+  templateUrl: './find-character-form.component.html',
+  styleUrl: './find-character-form.component.scss'
+})
+export class FindCharacterFormComponent implements OnDestroy {
+  static CharacterSearchStatusEnum = CharacterSearchStatusEnum;
+
+  get charSearchStatus() {
```

Comment:
why do you need a getter for enum with static data?

Reply:
I thought, that I need, but I couldn't explain why. I didn't understand how to use CharacterSearchStatusEnum in template. And you have already explained to me, that I don't need it, thank you)